lang/es.json

```diff
@@ -0,0 +1,955 @@
+            "content_count": "Noticias e información ({count})"
+        },
+        "faceted": {
+            "selected": {
+                "title": "Restringir por",
+                "rating-label": "Calificación de {rating, plural, one {# estrella} other {# estrellas}} o más",
+                "no-filters": "No se han aplicado filtros",
+                "clear-all": "Borrar todo"
+            },
+            "range": {
+                "update": "Actualizar",
+                "min-placeholder": "Mín.",
+                "max-placeholder": "Máx."
+            },
+            "rating": {
+                "and-up": "Y más"
+            },
+            "toggleSection": "Alternar sección del filtro {title}",
+            "clear": "Borrar",
+            "more": "Más",
+            "show-more": "Mostrar más",
+            "show-less": "Mostrar menos",
+            "show-filters": "Mostrar filtros",
+            "hide-filters": "Ocultar filtros",
+            "browse-by": "Buscar por"
+        },
+        "error": {
+            "empty_field": "El campo de búsqueda no puede estar vacío."
+        },
+        "tabs_accesibility_hint": "Use las flechas izquierda y derecha para pasar de una pestaña a otra."
+    },
+    "page_not_found": {
+        "page_heading": "404 Error - Página no encontrada",
+        "message": "Vaya, parece que la página que busca ha cambiado de ubicación o ya no existe."
+    },
+    "server_error": {
+        "page_heading": "Algo salió mal…",
+        "message": "La tienda no está disponible ahora. Inténtelo de nuevo en unos minutos."
+    },
+    "forbidden": {
+        "page_heading": "¡Lo sentimos! Inicie sesión para continuar",
+        "if_you_were_signed_in": "Si había iniciado sesión,",
+        "sign_in": "vuelva a hacerlo,",
+        "to_continue": "para reanudar su trabajo en una nueva sesión."
+    },
+    "sitemap": {
+        "page_title": "Mapa del sitio",
+        "show_all_link": "Mostrar todo"
+    },
+    "text_truncate": {
+        "view_more": "Ver más detalles",
+        "view_less": "Ocultar detalles"
+    },
+    "maintenance": {
+        "down": "Fuera de servicio por mantenimiento"
+    },
+    "admin": {
+        "maintenance_header": "Su tienda está cerrada por mantenimiento.",
+        "maintenance_tooltip": "Solo los administradores pueden ver esta tienda en este momento. Vaya a la página de configuración para desactivar el modo de mantenimiento.",
+        "maintenance_showstore_link": "Haga clic aquí para ver lo que verán sus visitantes.",
+        "prelaunch_header": "Su tienda en línea es privada. Comparta su sitio con un código de vista previa:",
+        "page_builder_link": "Diseñar esta página en Page Builder"
+    },
+    "carousel": {
+        "arrow_and_dot_aria_label": "Ir a diapositiva [NUMBER] de",
+        "active_dot_aria_label": "activo",
+        "content_announce_message": "Actualmente se encuentra en la diapositiva [NUMBER] de",
```

Comment:
the same here: `Actualmente se encuentra en la diapositiva [SLIDE_NUMBER] de [SLIDES_QUANTITY]`

Reply:
updated.